development/PIP_INSTALL.md

```diff
@@ -0,0 +1,149 @@
+# Python PIP Install Instructions
+
+Pacifica is highly recommended to use virtual environments to install manually.
+
+## Install Software Dependencies
+
+There are several external services that need to be installed and configured to use the Pacifica services.
+
+### PostgreSQL
+
+Download and install the latest version of PostgreSQL for your platform.
+
+For RedHat systems.
+
+```
+yum -y install postgresql-server
+service postgresql initdb
+service postgresql start
+```
+
+For Windows use EnterpriseDB installer.
+
+Adding users and databases require that the user be given all permissions to
+the database specified.
+
+ * Metadata service requires:
+  - Username is `pacifica`
+  - Password is `pacifica`
+  - Database is `pacifica_metadata`
+
+### MySQL
+
+Download and install MySQL (or equivalent) for your platform.
+
+```
+yum -y install mysql-server
+service mysql start
+```
+
+Adding users and databases require that the user be given all permissions to
+the database specified.
+
+ * Cart service requires:
+  - Username is `cartd`
+  - Password is `cartd`
+  - Database is `pacifica_cart`
+ * Ingest service requires:
+  - Username is `ingest`
+  - Password is `ingest`
+  - Database is `pacifica_ingest`
+ * UniqueID service requires:
+  - Username is `uniqueid`
+  - Password is `uniqueid`
+  - Database is `pacifica_uniqueid`
+
+### RabbitMQ
+
+Download and install the RabbitMQ server for your platform.
+
+For RedHat bases systems you should follow the RabbitMQ
+[install instructions](https://www.rabbitmq.com/install-rpm.html).
+
+You should also configure a user with access to vhosts as follows.
+
+ * Cart service requires:
+  - Username is `guest`
+  - Password is `guest`
+  - VHost is `/cart`
+ * Ingest service requires:
+  - Username is `guest`
+  - Password is `guest`
+  - VHost is `/ingest`
+
+### ElasticSearch
+
+Download and install ElasticSearch for your platform.
+
+For Redhat bases systems you should follow the Elasticsearch
+[install instructions](https://www.elastic.co/guide/en/elasticsearch/reference/current/rpm.html).
+
+
+### Install Python
+
+Install Python for your platform most Linux systems have this done already.
```

Comment:
Is this python 2 or 3?

Reply:
I put some info on this, Python 2.7 is recommended for now, Python 3 is in the works.